components/operator/config/crd/kustomization.yaml

```diff
@@ -2,34 +2,44 @@
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
-- bases/stack.formance.com_stacks.yaml
-- bases/stack.formance.com_configurations.yaml
-- bases/stack.formance.com_versions.yaml
-- bases/stack.formance.com_migrations.yaml
-- bases/formance.com_databases.yaml
-- bases/formance.com_stacks.yaml
-- bases/formance.com_brokertopics.yaml
-- bases/formance.com_gatewayhttpapis.yaml
-- bases/formance.com_ledgers.yaml
-- bases/formance.com_gateways.yaml
-- bases/formance.com_auths.yaml
-- bases/formance.com_authclients.yaml
-- bases/formance.com_wallets.yaml
-- bases/formance.com_orchestrations.yaml
-- bases/formance.com_webhooks.yaml
-- bases/formance.com_reconciliations.yaml
-- bases/formance.com_payments.yaml
-- bases/formance.com_searches.yaml
-- bases/formance.com_benthosstreams.yaml
-- bases/formance.com_benthos.yaml
-- bases/formance.com_stargates.yaml
-- bases/formance.com_versions.yaml
-- bases/formance.com_settings.yaml
-- bases/formance.com_resourcereferences.yaml
-- bases/formance.com_brokerconsumers.yaml
+  - bases/stack.formance.com_stacks.yaml
+  - bases/stack.formance.com_configurations.yaml
+  - bases/stack.formance.com_versions.yaml
+  - bases/stack.formance.com_migrations.yaml
+  - bases/formance.com_databases.yaml
+  - bases/formance.com_stacks.yaml
+  - bases/formance.com_brokertopics.yaml
+  - bases/formance.com_gatewayhttpapis.yaml
+  - bases/formance.com_ledgers.yaml
+  - bases/formance.com_gateways.yaml
+  - bases/formance.com_auths.yaml
+  - bases/formance.com_authclients.yaml
+  - bases/formance.com_wallets.yaml
+  - bases/formance.com_orchestrations.yaml
+  - bases/formance.com_webhooks.yaml
+  - bases/formance.com_reconciliations.yaml
+  - bases/formance.com_payments.yaml
+  - bases/formance.com_searches.yaml
+  - bases/formance.com_benthosstreams.yaml
+  - bases/formance.com_benthos.yaml
+  - bases/formance.com_stargates.yaml
+  - bases/formance.com_versions.yaml
+  - bases/formance.com_settings.yaml
+  - bases/formance.com_resourcereferences.yaml
+  - bases/formance.com_brokerconsumers.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
 patches:
+  - target:
```

Comment:
Why?

Reply:
I've changed that with `commonAnnotations` which does not override all annotations